lib/chai/utils/overwriteProperty.js

```diff
@@ -4,7 +4,9 @@
  * MIT Licensed
  */
 
+var chai = require('../../chai');
```

Comment:
I think it works because chai.js only sets `module.exports` and this file only calls stuff from chai.js inside of the methods, so it all kind of works out eventually.